src/server/bootstrap.rs

```diff
@@ -56,22 +56,29 @@ pub async fn init_hq_server(
 }
 
 pub async fn get_client_connection(server_directory: &Path) -> anyhow::Result<ClientConnection> {
+    let mut default_home = dirs::home_dir().unwrap_or_else(std::env::temp_dir);
+    default_home.push(".hq-server");
+
     let sd = ServerDir::open(server_directory).context("Invalid server directory")?;
     let access_record = sd.read_access_record().with_context(|| {
         format!(
-            "No running instance of HQ found (cannot read access record from {:?})",
+            "No running instance of HQ found (cannot read access record from {:?}), start server: <hq server start> or specify access record",
             sd.access_filename()
         )
     })?;
+    let server_dir_msg = if default_home != server_directory { format!("--server-dir {}", server_directory.to_str().unwrap()) } else { String::new()};
+
     let connection = HqConnection::connect_to_server(&access_record)
         .await
-        .with_context(|| {
+        .with_context(||
             format!(
-                "Cannot connect to HQ server at port {}:{}",
+                "Access token found but HQ server {}:{} is unreachable. Try to start the server using `hq server start {}`",
```

Comment:
Use `hq server start{}` and add the space conditionally to avoid unnecessary space.